test/controllers/PenaltyDetailsController.test.ts

```diff
@@ -29,7 +28,7 @@ describe('PenaltyDetailsController', () => {
 
 
             let session = createFakeSession([], config.cookieSecret, true);
-            session = session.saveExtraData(AppealKeys.APPEALS_KEY, appeal);
```

Comment:
I would not mind using some sort of constant for reading / writing an appeal from sessions extra data. Your change now increases chance of having typo like errors.